src/importers/baseImporter.ts

```diff
@@ -292,6 +293,7 @@ export abstract class BaseImporter {
         cipher.fields = [];
         cipher.login = new LoginView();
         cipher.type = CipherType.Login;
+        cipher.reprompt = CipherRepromptType.None;
```

Comment:
I believe it should be fine if we correctly handle Bitwarden exports correctly for now, not sure if it's the same mechanism as third party imports?